draft-fossati-tls-attestation.md

```diff
@@ -444,15 +456,16 @@ binder_collection = {
 {: #figure-tls-binder title="Format of TLS Binder Collection."}
 
 * Nonce is the value provided as a challenge by the relying party.
-* The identity key public fingerprint (ik_pub_fingerprint) is a SHA256 hash of
-  the Subject Public Key Info from the leaf X.509 certificate transmitted in the
-  handshake.
+* The identity key public fingerprint (ik_pub_fingerprint) is a hash of the
+  Subject Public Key Info from the leaf X.509 certificate transmitted in
+  the handshake.
 * The channel binder (channel_binder) is a partial transcript of the TLS
   handshake, up to (but not including) the Certificate message.
 
-A SHA256 digest of the encoded binder must be included in the attestation
-evidence.
+A hash of the encoded binder must be included in the attestation evidence.
 
+The hash algorithm negotiatied within the handshake must be used wherever
+hashing is required for the binder.
```

Comment:
we should probably note here that certain RoT APIs want their "challenge" of a certain size (TPM 2, PSA), which could imply truncation or expansion/padding.

Reply:
Added a note about that in the appendix.